### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by PrashantSakre**

### CODE REVIEW

File: static/java_script/ace-script.js [hunk 1]
```diff
@@ -217,19 +221,17 @@ function copy() {
...
-        document.body.removeChild(ele);
-    }
+    ele.setAttribute('download', "query.json");
+    document.body.appendChild(ele);
 
-    var text = document.getElementById("export-textarea-content").value;
-    download_file(text);
+    ele.click();
+    document.body.removeChild(ele);
 }
```
Comment (vivek-shrikhande) [inline static/java_script/ace-script.js]: Add an empty line at the end of file.

File: static/main.css [hunk 2]
```diff
@@ -222,12 +217,12 @@ body {
...
     text-align: end;
     margin: 0 1% 0 0;
-    padding-bottom: .1rem;
+    padding: 0 0 .1rem 0;
 }
 
-.footer-a {
+.footer a {
     font-size: 11px;
 }
```
Comment (vivek-shrikhande) [inline static/main.css]: Add an empty line at the end of file.

File: templates/home.html [hunk 3]
```diff
@@ -93,9 +93,9 @@
...
 <!-- Footer - section -->
-<div>
-    <p class="footer-p">
-        <a class="footer-a" href="https://ace.c9.io/" target="_blank" rel="noopener noreferrer">Editors powered by Ace</a>
+<div class="footer">
+    <p>
+        <a href="https://ace.c9.io/" target="_blank" rel="noopener noreferrer">Editors powered by Ace</a>
     </p>
 </div>
 {% endblock content %}
```
Comment (vivek-shrikhande) [inline templates/home.html]: Add an empty line at the end of file.

**Review by vivek-shrikhande — CHANGES_REQUESTED** [review]

File: static/main.css (hunk 2 above)
Comment (PrashantSakre) [reply static/main.css]: Added..

**Review by vivek-shrikhande — APPROVED** [review]

**Pull request merged**